kernel/main.cpp

```diff
@@ -118,22 +118,24 @@ extern "C" void px_kernel_main(const multiboot_info_t* mb_struct, uint32_t mb_ma
     px_kprintf(DBG_OKAY "Done.\n");
     px_kernel_boot_tone();
     // Keep the kernel alive.
-    px_kprintf("\n");
     int i = 0;
     while (true) {
         // Display a spinner to know that we're still running.
+        // Each line will save the cursor position, change it
+        // to the top right corner (-1) and then restore the
+        // cursor position (to the saved location).
         switch (i) {
             case 0:
-                px_kprintf("\b|");
+                px_kprintf("\033[s\033[%i;%iH\b|\033[u", X86_IND_Y, X86_IND_X);
```

Comment:
I understand wanting to test with this, but maybe it should be merged in separately once this PR is ready?

Reply:
Fair enough. I'll remove it and the makefile change.